baybe/parameters/custom.py

```diff
@@ -88,6 +89,13 @@ def _validate_custom_data(  # noqa: DOC101, DOC103
                 f"The custom dataframe for parameter {self.name} has columns "
                 "that contain only a single value and hence carry no information."
             )
+        if value.duplicated().any():
+            raise ValueError(
+                f"The custom dataframe for parameter {self.name} has duplicated rows."
+                f"This is not allowed because it creates problems with the "
```

Comment:
ive changed the wording to `supported` but having more explanation is tricky, its beuacse of the merge we are force to do because botorch returns the optimzied points and not the indices, do you want to open this keg here by mentioning that in this error message?

Reply:
good suggestion, added
![image](https://github.com/user-attachments/assets/bd0bdb5f-32bf-47cc-b313-65d6f45b212a)